docs/src/main/tut/datatypes/either.md

```diff
@@ -212,132 +212,6 @@ magic("123") match {
 }
 ```
 
-## Either in the small, Either in the large
```

Comment:
Please be more careful about removing this section - variance does not subsume the concerns here.